### ISSUE TITLE
Please make a release

### ISSUE THREAD

**Issue by yurivict** [openbabel/openbabel#1936]
I tried to update the FreeBSD port to the latest revision, but 7 other ports fail to build with the latest OpenBabel version.

v_sim fails:
```
In file included from /usr/local/include/openbabel-2.0/openbabel/obconversion.h:38:
/usr/local/include/openbabel-2.0/openbabel/lineend.h:74:31: warning: unused parameter 'which' [-Wunused-parameter]
      std::ios_base::openmode which = std::ios_base::in | std::ios_base::out )
                              ^
/usr/local/include/openbabel-2.0/openbabel/lineend.h:84:31: warning: unused parameter 'which' [-Wunused-parameter]
      std::ios_base::openmode which = std::ios_base::in | std::ios_base::out )
                              ^
ob_basic.cpp:93:17: error: use of undeclared identifier 'etab'
  return (char*)etab.GetSymbol(atom->GetAtomicNum());
                ^
```

ghemical fails:
```
filetrans.cpp:53:9: error: unknown type name 'OBElementTable'
        extern OBElementTable etab;
               ^
filetrans.cpp:56:9: error: unknown type name 'OBIsotopeTable'
        extern OBIsotopeTable isotab;
               ^
```

Also avogadro, osra, kalzium, molsketch, xdrawchem fail.

You need to make an official release candidate, and notify these packages' maintainers to make them compatible.

You can't just keep developing, gradually deviating from everything else, because people won't be able to use OpenBabel at all due to these incompatibilities.

**Comment by baoilleach** [comment]
The next version breaks the API...for the first time in more than a decade. It's not a gradual deviation, it's a single sharp shock that allows us to address some really fundamental issues that were causing problems. But it's taking a while, because we don't have a lot of free time. So a week or two of development is spread over maybe half a year.

**Comment by baoilleach** [comment]
On the other hand, I should say that I *am* working towards a release. I just need to tidy up a few things.

**Comment by afonari** [comment]
Any updates on this?

**Comment by JohnScott623** [comment]
3.0 was released in October, so this issue can be closed.

**Comment by ghutchis** [comment]
I'll say for the record that older packages probably won't update to the 3.0 api. For example, I have no desire to fix Avogadro 1.2, since effort is going into getting `avogadrolibs` released as 2.0.
